scripts/cnn_variant_wdl/vcf_analysis_no_truth.R

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env Rscript
```

Comment:
I don't love that there's so much duplication here.  Can you call an R script from another R script from the command line?

Reply:
Same here -- if this is too painful then forget it.